src/docs/contributing/platforms/platform-location.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Platform Delineation
+noindex: true
+---
+
+We separate platform portals by the main way or mechanism that developers interact with the platform and with Sentry.
```

Comment:
should this just go in the general "how do platform portals work" ?

Reply:
Yep, that's better.